packages/filebrowser-extension/src/index.ts

```diff
@@ -162,6 +162,12 @@ const browser: JupyterFrontEndPlugin<void> = {
       restorer.add(browser, namespace);
     }
 
+    // Navigate to preferred-dir trait if found
+    const preferredPath = PageConfig.getOption('preferredPath');
+    if (preferredPath) {
```

Comment:
Note that in the case of an exception in the jupyterlab_server case (missing trait), preferredPath will be "/", which will trigger this branch, causing a cd. I'm curious how this cd interacts with the tree router? I.e. If I specify a path in the URL via /tree, what will happen? Will this cause an additional call to the contents API? Which folder will end up being the final one?

Reply:
Hi @vidartf, so since the browser works on relatives paths dong a `cd('/')` will just go to the root path, which is the default behavior anyway.